.github/workflows/crates-release.yml

```diff
@@ -0,0 +1,30 @@
+# See https://crates.io/docs/trusted-publishing
+name: Publish to crates.io
+on:
+  workflow_dispatch:
+    inputs:
+      crate:
+        description: 'Crate to publish'
+        required: true
+        type: choice
+        options:
+          - bootc-internal-utils
```

Comment:
Combining with the previous I think we can just publish all the relevant crates on git tags.